src/trufont/objects/factories.py

```diff
@@ -97,6 +97,20 @@ def pathGraphicsPathFactory(path):
             if point.type == "curve":
                 graphicsPath.AddCurveToPoint(*stack)
                 stack = []
+            elif point.type == "qcurve":
+                if len(stack) == 4:
+                    graphicsPath.AddQuadCurveToPoint(*stack)
+                else:
+                    # Group off-curves into quads to construct implied on-curve points.
```

Comment:
"Group off-curves into quads" it's more like splitting than grouping, but I would say "make simple quads by constructing implied oc" because that's what's happening. large segment broken down to simple ones

Reply:
I don't understand that sentence 😬

Reply:
By constructing implied oc (`(cx1 + cx2) / 2, (cy1 + cy2) / 2`) we can make simple quads (a segment made of just one off followed by on which is the canonical form)

Reply:
Oh, ok. I thought by "quad" you meant the four elements in the stack slice.